skin-resque-api/init-mongo.js

```diff
@@ -0,0 +1,549 @@
+db = new Mongo().getDB("skin-resque");
+
+db.createCollection('Cosmetics', {capped: false});
+
+db.Cosmetics.insert([
```

Comment:
I think that web scraper should insert data to the database

Reply:
This is adding data by docker-compose